docker/images/mainnet/README.md

```diff
@@ -0,0 +1,43 @@
+# Docker Build
```

Comment:
Please use Docker library documentation structure (e.g. [`library/mongo` README.md](https://github.com/docker-library/docs/tree/master/mongo/README.md) ) as a reference for writing image docs file .